draft-thomson-webpush-protocol.xml

```diff
@@ -655,19 +586,18 @@ DATA         [stream 4] +END_STREAM
           include the "private" directive.
         </t>
         <t>
-          A push service can invalidate a registration or subscription at any
-          time.  If a user agent has an outstanding request to the
-          "registration" resource (see <xref target="monitor"/>), this can be
-          signaled by returning a 400-series status code, such as 410 (Gone).  A
-          push service uses server push to indicate that a subscription has
-          expired; a pushed 400-series status code for the subscription resource
-          signals the termination of a subscription.
+          A push service can invalidate a subscription at any time. If a user
+          agent has an outstanding request to the "monitor" resource (see <xref
+          target="monitor"/>), this can be signaled by returning a 400-series
+          status code, such as 410 (Gone).  A push service uses server push to
+          indicate that a subscription has expired; a pushed 400-series status
+          code for the "monitor" resource signals the termination of a
+          subscription.
         </t>
         <t>
-          A user agent can request that a registration or subscription be
+          A user agent can request that a subscription be
           removed by sending a DELETE request to the corresponding URI.
-          Removing a registration causes all associated subscriptions to be
-          removed.  Requests to delete a subscription MUST be authenticated; the
+          Requests to delete a subscription MUST be authenticated; the
           URI for subscriptions will be known by multiple entities, most of whom
           will not be authorized to remove the subscription.
```

Comment:
This is also no longer true.  The "monitor" URI will not be known to any entity other than the user agent, therefore knowledge of the monitor URI could be sufficient to authorize deletion of a subscription.